api/src/client/dynamics/housing/DynamicsHousingRegistrationClient.ts

```diff
@@ -53,7 +62,7 @@ export const DynamicsHotelMotelRegistrationClient = (
           }
           return 1;
         });
-        return sortedRegistrations;
+        return sortedRegistrations.slice(0, 1);
```

Comment:
For this iteration, we decided to only show the most recent registration. However, I'm keeping this as an array since we have talked about wanting to show history again in the future